tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/jvm/TypesTest.java

```diff
@@ -572,8 +572,9 @@ public void testJsonArrayToJsonCasting() {
         Assert.assertEquals(returns[0].stringValue(), "[[1, 2, 3], [3, 4, 5], [7, 8, 9]]");
     }
 
-    @Test(expectedExceptions = { BLangRuntimeException.class },
-            expectedExceptionsMessageRegExp = ".*incompatible types: 'error' cannot be cast to 'string'.*")
+    @Test(expectedExceptions = {BLangRuntimeException.class},
+            expectedExceptionsMessageRegExp = ".*incompatible types: 'error' cannot be cast to 'string'.*",
+            enabled = false)
```

Comment:
This is the original source function which is tested here.
```
function testGetFromNull () returns (string) {
    json j2 = {age:43, name:null};
    string value = <string>j2.name.fname;
    return value;
}
```
With this change, I was unable to reproduce the expected error. Therefore it is disabled. Any suggestions to refactor this test case?

Reply:
Based on the name of the test seems like we're trying to test the behaviour of field access on null, so shouldn't we change the source to something like

```ballerina
function testGetFromNull () {
    json j2 = {age:43, name:null};
    json value = checkpanic j2.name.fname;
}
```

and use `expectedExceptions` here?

Reply:
Also, although not applicable here, IMO if there are tests that are no longer valid we should remove them instead of disabling them. Or it will be hard to track why something is disabled in the long term and what needs to be enabled.

Reply:
This example does not throw the expected exception. That test was removed.